config/settings.yml

```diff
@@ -40,6 +41,8 @@ SCIENCEWIRE:
   PORT: 443
   RECOMMENDATION_PATH: /PublicationCatalog/MatchedPublicationItemIdsForAuthor?format=xml
   TMPDIR: /tmp
+  ARTICLE_BASE_URL: https://ws.isiknowledge.com/cps/openurl/service?url_ver=Z39.88-2004&rft_id=info:ut/
```

Comment:
I think so, the url only works when you append the appropriate specific article ID in the code.